Makefile

```diff
@@ -55,31 +48,35 @@ SYSCONFDIR:=$(DEFAULT_SYSCONFDIR)
 endif
 SYSCONFDIR:=$(subst PREFIX,$(PREFIX),$(SYSCONFDIR))
 
-export APPLY_PATCHES
+ifndef NEW_BUILD
+NEW_BUILD=$(DEFAULT_NEW_BUILD)
+endif
+
 export VERSION
 export DESTDIR
 export PREFIX
 export SYSCONFDIR
 export SKIP_TESTS
+export NEW_BUILD
 
 all: install
 
 
 archive: install
+ifeq ($(NEW_BUILD),no)
 	rm -f $(CURDIR)/$(PACKAGE_NAME).tar.gz && cd $(DESTDIR) && tar -czf $(CURDIR)/$(PACKAGE_NAME).tar.gz $(PREFIX)
 	rm -f $(CURDIR)/$(PACKAGE_NAME).zip && cd $(DESTDIR) && zip -r $(CURDIR)/$(PACKAGE_NAME).zip $(PREFIX)
-
-apply-patches: $(wildcard patches/*)
-ifeq ($(APPLY_PATCHES),yes)
-	git reset --hard HEAD
-	cat patches/series | xargs -iPATCH bash -c 'patch -p1 < patches/PATCH'
 endif
 
-build: apply-patches
+build:
+ifeq ($(NEW_BUILD),yes)
+	pip3 install requests
+	python3 download_artifact
```

Comment:
i think we're missing the `download_artifact` script?

Reply:
I actually put the scripts in the master branch because I think it can be reused later by other branches. However, just for archive, I should move the scripts on this branch.

Reply:
ah, i see, link relevant PR? anyone looking at history will appreciate the connection since it's hard to reason about PRs that are dependent on each other without links.

additionally, in cases like this it is helpful to link the other PR since the other PR needs to be merged *before* this one is merged. understanding merge dependency order also helps with PR review and allows us to make sure we merge in the right order